pkg/ddc/alluxio/health_check.go

```diff
@@ -322,3 +323,101 @@ func (e *AlluxioEngine) checkFuseHealthy() (err error) {
 	}
 	return err
 }
+
+
+// CheckExistenceOfEngine check engine existed
+func (e *AlluxioEngine) CheckExistenceOfEngine() (err error) {
+
+	master, masterErr := kubeclient.GetStatefulSet(e.Client, e.getMasterName(), e.namespace)
```

Comment:
I'm wondering the reason you don't put this logic into checkMasterHealthy, checkWorkerHealthy, checkFuseHealthy separately?